vsi2tif/src/process.py

```diff
@@ -48,7 +49,10 @@ def cellsens2tif_batch(
     # perform conversion in separate processes
     for root, file in tqdm(paths):
         curr_input_path = os.path.join(root, file)
-        curr_output_path = (output_path + "/" + curr_input_path.split(input_path)[-1]).replace(".vsi", ".tif")
+        if remove_name_spaces:
```

Comment:
Isn't it cleaner to do the `.replace(" ", "_")` update in a newline after the `curr_output_path` is created, only if `remove_name_spaces` is True? That way, you can remove the else statement.

Reply:
@dbouget Can you address this?

Reply:
If we do it that way over the whole curr_output_path, it will also remove spaces from folder names (if some people have that), so that's maybe not a good idea?

Reply:
That makes sense. Then this is OK!